roles/couchdb-lucene/defaults/main.yml

```diff
@@ -4,5 +4,5 @@ couchdb_lucene_system_group: "root"
 couchdb_lucene_bind_host: "localhost"
 couchdb_lucene_bind_port: 5985
 couchdb_lucene_java_opts: "-server -Xmx1g -XX:OnOutOfMemoryError=bin/kill_ppid"
-couchdb_lucene_local_checkout_path: "/mnt/vol1/couchdb-lucene"
-couchdb_lucene_local_package_directory: "/mnt/vol1/couchdb-lucene-tars"
\ No newline at end of file
+couchdb_lucene_local_checkout_path: "/tmp/couchdb-lucene"
```

Comment:
Makes sense.

In Ona's private inventories, could you please change these to `/tmp/{{ lookup('env','USER') }}/couchdb-lucene` and `/tmp/{{ lookup('env','USER') }}/couchdb-lucene-tars` to avoid permission issues when one person runs the playbook then all other users are unable to read the files.